sktime/transformations/panel/summarize/_extract.py

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8 -*-
-"""Sequence feature extraction transformers."""
+"""Sequence feature extraction transformers.
+
+These transformers expect a dataframe of series, usually univariate and
```

Comment:
please remove these three lines - this is true for all transformers

Reply:
done in cd28ff93